gcp/appengine/frontend_handlers.py

```diff
@@ -92,6 +94,21 @@ def add_links(bug):
       affected['fixed_link'] = _commit_to_link(repo_url, affected['fixed'])
 
 
+def add_source_info(bug, response):
+  """Add source information."""
```

Comment:
Maybe "Add source information to |response|." so that it's clearer this mutates response.

Reply:
done!